### PR TITLE
feat(ComponentReact): re-enable React component example page

### LINKED ISSUE #803: Add React component examples for Dropdown to website
Add React examples to website for Dropdown. (see current site for examples https://www.carbondesignsystem.com/components/dropdown/code) Could use codesandbox plugin for this. 

- [x] Multiselect Dropdown
- [x] Inline Multiselect Dropdown
- [x] Filter Dropdown

### PR DESCRIPTION
**Pull request by asudoh**
Refs #803.

#### Changelog

**New**

- `code.md` files for multi select and combo box.

**Changed**

- Tweaks to `<component-react>` to catch up with the latest of Storybook and live demo scenarios.
- Tweak to "View on React Storybook" style so its z-order is under dropdown box.

### CODE REVIEW

**Comment by asudoh** [comment]
@alisonjoseph Good catch, made an update to address this - Please unregister the service worker before trying out the new code. Thanks!

**Comment by lovemecomputer** [comment]
i see some unfortunate alignment on these component examples : (
screen shot 2019-02-11 at 11 47 05 am

**Comment by asudoh** [comment]
@lovemecomputer Good point! - The overlapping thing is tracked in: IBM/carbon-components#1031.

**Comment by aagonzales** [comment]
Let's keep them all one page for now since all of the usage content for each is on the main dropdown page.

**Comment by alisonjoseph** [comment]
@asudoh can you move these to the Dropdown page 🙏

**Comment by asudoh** [comment]
@alisonjoseph Done - Also upgraded `carbon-components` to address the icon overlap issue.

**Comment by jnm2377** [comment]
This is awesome! I noticed that the icons these examples are using aren't the v10 icons like [here](http://react-experimental.carbondesignsystem.com/?selectedKind=ComboBox&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel).
screen shot 2019-02-13 at 3 25 22 pm
screen shot 2019-02-13 at 3 25 42 pm

Also it looks like it needs more spacing between the header and subtitle text for each example as well as between each example (at least in the Netlify link, when I view locally I'm getting a 404 for MultiSelect), but Idk what exact spacing design would want.
screen shot 2019-02-13 at 3 23 43 pm

And there's a small typo in the bottom right link of filterable multi select. It says "filtable".

Also, I know this isn't coming from the website repo, but at some point we should update the documentation for these components cause it seems like something is going wrong for some of them. This one looks like it's missing an image or something.

screen shot 2019-02-13 at 3 34 35 pm

File: src/components/ComponentReact/ComponentReact.js [hunk 1]
```diff
@@ -70,14 +76,22 @@ class ComponentReactExample extends Component {
...
-    } else {
-      ReactDOM.render(<NewComponent />, this.comp);
     }
   };
 
   render() {
     const { name, component, variation } = this.props;
-    const componentLink = `http://react.carbondesignsystem.com/?selectedKind=${component}&selectedStory=${variation}`;
+    const storybookMessage = {
+      'MultiSelect-MultiSelect.Filterable': 'Check off Filtable in KNOBS tab',
```
Comment (jnm2377) [inline src/components/ComponentReact/ComponentReact.js]: Should this be  'Filterable' ?

Comment (asudoh) [reply src/components/ComponentReact/ComponentReact.js]: Good catch @jnm2377! Fixed.

**Comment by asudoh** [comment]
Updated the WebPack config to utilize feature flags so React components live demo code picks up experimental icon. Also the markup/styles of `<ComponentReact>` closer to its vanilla counterpart for better styling.

**Comment by jnm2377** [comment]
For some reason the MultiSelect is still giving me a 404, both locally and in the netlify preview. The other component pages look great! Those style updates and icons made a huge difference 👍 🎉

**Comment by alisonjoseph** [comment]
Looking good, noticed this issue, might be an upstream issue though in components?

screen shot 2019-02-14 at 12 18 38 pm

Also, did we want to remove the links/pages for combo box and multi select since they are on the dropdown page? (looks like we still need to add combo box to dropdown maybe?

**Comment by jeanservaas** [comment]
Yes @alisonjoseph I think we remove those links from the nav since they are on the Dropdown page... that how it was previously @aagonzales thinks so too.

**Comment by asudoh** [comment]
Thanks @jnm2377 and @alisonjoseph for additional review - Fixed the navigation and merged combo box page to dropdown page.

> Looking good, noticed this issue, might be an upstream issue though in components?

> screen shot 2019-02-14 at 12 18 38 pm

If you mean the bottom border, yes, upstream has the same behavior. Do we want the border like what combo box shows?

**Comment by tw15egan** [comment]
@carbon-design-system/design is this good to go?

**Comment by jeanservaas** [comment]
Hey guys, not quite sure what I'm reviewing here, just went to the dropdown page and am still seeing issues with dropdown, filterable multi-select and combo box ... 

Noticed @alisonjoseph said above, these upstream issues shouldn't stop this from getting merged, so is this issue just about IA and where to put these react components? 

image

image

image

**Comment by asudoh** [comment]
Thanks @jeanservaas for reviewing!

> so is this issue just about IA and where to put these react components?

Yes if I understand what you meant by IA correctly. In other words, this how each components look goes beyond the purview of this PR. But please don’t hesitate to enter component UI ones to `carbon-components` repo especially if no existing issue cover them!

**Pull request merged**